submission/programs/adae.R

```diff
@@ -0,0 +1,283 @@
+# Name: ADAE
+#
+# Label: Adverse Event Analysis Dataset
+#
+# Input: ae, adsl
+# Output: adae.xpt
+#
+# Developed by : [Author/Developer name]
+
+library(admiral)
+library(dplyr)
+library(tidyr)
+library(lubridate)
+library(stringr)
+library(diffdf)
+library(xportr)
+library(metacore)
+library(metatools)
+
+# ----------------------------------------------------------------------------#
+# Use e.g. haven::read_sas to read in .sas7bdat, or other suitable functions  #
+# as needed and assign to the variables below.                                #
+# For illustration purposes read in admiral test data                         #
+# ----------------------------------------------------------------------------#
+
+# ---------- #
+# read in AE #
+# ---------- #
+ae <- haven::read_xpt(file.path("sdtm", "ae.xpt"))
```

Comment:
We should be consistent with prefix. Or we do it everywhere or only when absolutely required.

Reply:
The latter please so let's remove `haven::`.

Reply:
Removed.

Reply:
@SHAESEN2 or @thomas-neitmann If ok, please `Resolve conversation`.